tx/textual/valuerenderer/coins.go

```diff
@@ -67,7 +66,7 @@ func (vr coinsValueRenderer) Format(ctx context.Context, v protoreflect.Value, w
 				return err
 			}
 
```

Comment:
Why aren't we calling `NewCoins`?

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/12729#discussion_r961194673

Reply:
Ok, so it's to not import all the methods and depend on root huh?

Reply:
Yeah, root depending on go modules is fine (e.g. `math` and `error`) but the other way round no.